azure/terraform/appservice.rego

```diff
@@ -667,4 +667,361 @@ app_service_remote_debugging_disabled_metadata := {
+}
+
+
+#
+# PR-AZR-0080-TRF
+#
+
+default app_service_dot_net_framework_latest = null
+
+azure_attribute_absence["app_service_dot_net_framework_latest"] {
+    resource := input.resources[_]
+    lower(resource.type) == "azurerm_app_service"
+    not resource.properties.site_config
+}
+
+#Defaults to v4.0
+azure_attribute_absence["app_service_dot_net_framework_latest"] {
+    resource := input.resources[_]
+    lower(resource.type) == "azurerm_app_service"
+    site_config := resource.properties.site_config[_]
+    not site_config.dotnet_framework_version
+}
+
+azure_issue["app_service_dot_net_framework_latest"] {
+    resource := input.resources[_]
+    lower(resource.type) == "azurerm_app_service"
+    site_config := resource.properties.site_config[_]
+    lower(site_config.dotnet_framework_version) != "v6.0"
+}
+
+# we need to make it pass if property is missing, as azurerm_app_service may not need dot net framework
+app_service_dot_net_framework_latest {
+    azure_attribute_absence["app_service_dot_net_framework_latest"]
+    not azure_issue["app_service_dot_net_framework_latest"]
+}
+
+app_service_dot_net_framework_latest {
+    lower(input.resources[_].type) == "azurerm_app_service"
+    not azure_attribute_absence["app_service_dot_net_framework_latest"]
+    not azure_issue["app_service_dot_net_framework_latest"]
+}
+
+app_service_dot_net_framework_latest = false {
+    azure_issue["app_service_dot_net_framework_latest"]
+}
+
+app_service_dot_net_framework_latest_err = "Azure App Service currently dont have latest version of Dot Net Framework" {
+    azure_issue["app_service_dot_net_framework_latest"]
+}
+
+app_service_dot_net_framework_latest_metadata := {
+    "Policy Code": "PR-AZR-0080-TRF",
+    "Type": "IaC",
+    "Product": "AZR",
+    "Language": "Terraform",
+    "Policy Title": "Azure App Service Dot Net Framework should be latest",
+    "Policy Description": "This policy will identify the Azure app service which dont have latest version of Dot Net Framework and give alert",
+    "Resource Type": "azurerm_app_service",
+    "Policy Help URL": "",
+    "Resource Help URL": "https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/app_service"
+}
+
+
+#
+# PR-AZR-0081-TRF
+#
+
+default app_service_php_version_latest = null
+
+azure_attribute_absence["app_service_php_version_latest"] {
+    resource := input.resources[_]
+    lower(resource.type) == "azurerm_app_service"
+    not resource.properties.site_config
+}
+
+azure_attribute_absence["app_service_php_version_latest"] {
+    resource := input.resources[_]
+    lower(resource.type) == "azurerm_app_service"
+    site_config := resource.properties.site_config[_]
+    not site_config.php_version
+}
+
+azure_issue["app_service_php_version_latest"] {
+    resource := input.resources[_]
+    lower(resource.type) == "azurerm_app_service"
+    site_config := resource.properties.site_config[_]
+    to_number(site_config.php_version) != 7.4
```

Comment:
move the php version to a variable in the rego

Reply:
fixed.